src/csharp/Microsoft.Spark.Worker.UnitTest/DaemonWorkerTests.cs

```diff
@@ -0,0 +1,135 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Spark.Interop.Ipc;
+using Microsoft.Spark.Network;
+using Razorvine.Pickle;
+using Xunit;
+
+namespace Microsoft.Spark.Worker.UnitTest
+{
+    public class DaemonWorkerTests
+    {
+        [Theory]
+        [InlineData(1)]
+        [InlineData(3)]
+        public void TestsDaemonWorkerTaskRunners(int taskRunnerNumber)
+        {
+            ISocketWrapper daemonSocket = SocketFactory.CreateSocket();
+            var typedVersion = new Version(Versions.V2_4_0);
+            var daemonWorker = new DaemonWorker(typedVersion);
+            
+            Task.Run(() => daemonWorker.Run(daemonSocket));
+
+            for (var i = 1; i <= taskRunnerNumber; i++)
+                CreateAndVerifyConnection(daemonSocket);
```

Comment:
Can you review https://github.com/dotnet/spark/blob/master/docs/coding-guidelines/csharp-coding-style.md?